gutsy-gamblers/main.kv

```diff
@@ -79,4 +80,42 @@
 
 		Button:
 		    text: "Revert date"
-		    on_press: root.revert_date()
\ No newline at end of file
+		    on_press: root.revert_date()
+
+<SettingsScreen>:
```

Comment:
Maybe add `title: "Settings"` as a property since it says "No title" at top without this.